src/test/regress/sql/upgrade_distributed_function_after.sql

```diff
@@ -0,0 +1,11 @@
+SET search_path TO upgrade_distributed_function_before, public;
+
+-- make sure that the metadata synced
+SELECT bool_and(metadatasynced) FROM pg_dist_node WHERE isactive AND noderole = 'primary';
+SET client_min_messages TO DEBUG1;
+
+-- these are simple select functions, so doesn't have any
+-- side effects, safe to be called without BEGIN;..;ROLLBACK;
+SELECT count_values(11);
+SELECT count_values(12);
+
```

Comment:
When do we delete the schema `upgrade_distributed_function_before` that we create in `upgrade_distributed_function_before.sql`? Shouldnt we delete it here? If so, we should probably rename it:

```sql
CREATE SCHEMA upgrade_distributed_function;
```